setup.py

```diff
@@ -34,12 +34,14 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: Implementation :: CPython",
     ],
     packages=find_packages('src'),
     include_package_data=True,
     package_dir={'': 'src'},
     namespace_packages=['plone', 'plone.recipe'],
+    python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*',
```

Comment:
I think we do not support Python 3.5?
```suggestion
    python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*,!=3.5.*',
```

Reply:
What is the reason for not supporting Python 3.5? Zope 4 supports it, so that means the recipe wouldn't fully support Zope 4?

Reply:
Afaik Plone never supported Python 3.5.

Reply:
Correct. But if this recipe is useful for Zope 4 users, then we can support, even though there is `plone` in the recipe name.

Reply:
@jensens I dove into the package history and noticed Python 3.5 support was removed for no pressing reason, most likely because Plone doesn't support it. But this recipe is supposed to build Zope 4 and 5 instances, and for that reason I re-added the claimed Python 3.5 support and would like to keep it in unless and until it's causing a support problem.

The tests run fine locally using `tox`, I only saw one issue in the GitHub actions tests that appear to come from `AccessControl`. For some reason during dependencies collection it wants to install `funcsigs` as dependency of `AccessControl`, which is clearly marked as only required for Python 3.3 and lower. So that makes no sense. I removed the GH actions test.